src/api/app/controllers/request_controller.rb

```diff
@@ -136,6 +136,24 @@ def destroy
     render_ok
   end
 
+  def request_status
```

Comment:
A bunch of nitpicks:

- `status` instead of `request_status` (we are already in `request_controller.rb`
- `BsRequest.find_by_number!(@id)`
- Better variable names like `source_project`, `target_project` and `source_package`